### PR TITLE
Update website after grand opening

### PR DESCRIPTION
**Pull request by danielabrozzoni**

### CODE REVIEW

**Review by casungo — APPROVED** [review]
LGTM!

File: content/it/_index.md [hunk 1]
```diff
@@ -5,28 +5,25 @@ menu: "main"
...
 
-{{< /framed >}}
+Siamo finanziati attraverso contribuzioni volontarie da individui e aziende private: se vuoi supportarci, considera di [fare una donazione]({{< ref "donate" >}}).
+La nostra pagina di [trasparenza]({{< ref "transparency" >}}) mostra nel dettaglio come gestiamo i fondi raccolti.
 
-## Fondatori
+## Community
 
-hack.bs è la creazione di Alekos Filini e Daniela Brozzoni, due sviluppatori Bitcoin che al momento lavorano alla libreria open-source [Bitcoin Dev Kit](https://bitcoindevkit.org).
+La nostra community è formata da circa 30 membri, tra cui sviluppatori, ingegneri, designer e studenti.
```
Comment (parsevalbtc) [inline content/it/_index.md]: Terrei "ricercatori" ;)

File: content/en/_index.md [hunk 2]
```diff
@@ -5,28 +5,26 @@ menu: "main"
...
-hack.bs is the creation of Alekos Filini and Daniela Brozzoni, two Bitcoin developers currently working on the open-source library [Bitcoin Dev Kit](https://bitcoindevkit.org).
+We regularly host workshops, meetups, hackathons and more, open to everybody. If you want to stay up to date you can subscribe to our [mailing list]({{< ref "mailing-list" >}}), or follow us on [Twitter](https://twitter.com/h4ckbs) and [Instagram](https://instagram.com/h4ckbs).
+
+We are funded through voluntary contributions from individuals and private companies: if you want to support our project consider [making a donation]({{< ref "donate" >}}).
+We have a [transparency]({{< ref "transparency" >}}) page to show exactly how we allocate our funds.
 
 ## Community
 
-We currently have around 20 members eagerly waiting for the hackerspace to open. 
+Our community is currently composed of 30-ish people, including developers, engineers, designers, students, and all-rounders. We're always excited to welcome new talent - whether you're an expert or just starting out, come say hi!
```
Comment (parsevalbtc) [inline content/en/_index.md]: Would keep "Researchers"

**Review by parsevalbtc — CHANGES_REQUESTED** [review]

**Comment by parsevalbtc** [comment]
Per il resto ACK, bel update @danielabrozzoni!

**Pull request merged**